app/models/like.rb

```diff
@@ -0,0 +1,4 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, counter_cache: :likes_counter
+end
```

Comment:
- [ ] Like model has a method that updates the likes counter for a post.

As mentioned above you can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #update_likes_counter

Reply:
Thanks for the observation! I have already applied the requested changes.